### ISSUE TITLE
Using vcluster higher than host cluster

### ISSUE THREAD

**Issue by anuragagarwal561994** [loft-sh/vcluster#1245]
### Is your feature request related to a problem?

I would like to use features such as SidecarContainers for my test setup, but as of now I can't upgrade my host cluster to 1.28.

As far as I understand since the pods are scheduled on the host cluster, I am able to create pods with vcluster using the right parameters in values.yaml but they are not getting scheduled on the host cluster because it doesn't support this feature.

### Which solution do you suggest?

Can there be an isolation mode where the scheduled pods also are taken care of by the vcluster deployed cluster.

**Comment by ishankhare07** [comment]
Hi @anuragagarwal561994 your pods always run on the host cluster. If your host cluster doesn't support that feature, there's conthing that vclsuter can do about it. Also you would need to enable the featuregate for this to work.

I recommend creating a kind cluster with the feature gate enabled and then use it as the host cluster to test this from vcluster.

**Comment by anuragagarwal561994** [comment]
Thanks @ishankhare07 for the quick response on this issue, yes I suspected that this won't be possible with vcluster.

Correct my theory if wrong, since vcluster is using the same nodes as the host cluster, the host clusters are having kubelets which are not supporting the new kubernetes features.

I tested on my local with k3s cluster with 1.28 in this it works properly, but I wanted to check if my production cluster is not upgraded can I use an alternate for a less critical feature.

Note for others if it is not working on your local, use the latest beta version of image (not to be used on production) 

```
syncer:
  image: ghcr.io/loft-sh/vcluster:0.16.0-beta.5
```

since this version uses the 1.28.1 API of kubernetes, if you are using the current released version of vcluster you won't be able to spawn the pod with sidecar containers enabled.
